cfgov/teachers_digital_platform/models/activity_index_page.py

```diff
@@ -349,6 +352,7 @@ def update_cards(self):
                 'summary': activity.summary,
                 'topic': activity.get_topics_list(),
                 'activity_duration': activity.activity_duration.title,
+                'available_in_spanish': 'Available in Spanish' if es else None,
```

Comment:
```suggestion
                'available_in_spanish': True if es else False,
```

If we pass this as a Boolean, then we can enter a translatable string in the template (`{{ _("Available in Spanish") }}`) which can itself be automatically translated into Spanish if a user has their browser's language set to Spanish, by including a `<span lang="es">` wrapper around the translated string. I think.

See https://cfpb.github.io/consumerfinance.gov/translation/ for more about how we do translations.

Even if the translation thing doesn't work, I think passing the boolean and having the string there in the template is a little cleaner.

Reply:
Thanks, Scott. I'll refactor this.

Reply:
Thanks for making that translatable. Could you please follow the steps on that doc page I linked to add the translated string to the list of available translations?

Reply:
> Thanks for making that translatable. Could you please follow the steps on [that doc page](https://cfpb.github.io/consumerfinance.gov/translation/) I linked to add the translated string to the list of available translations?

@nbucknor Re-upping this comment you may have missed.

Reply:
@Scotchester I've run `manage.py makemessages -l es` and committed the updated django.po file. However, when I run `manage.py compilemessages` and commit the updated django.mo file, I get failed translation unit tests in travis CI. Am I missing a step?

Reply:
@nbucknor Strange stuff! It appears that you reverted the committing of `django.mo` on this branch, correct? If I pull this branch down in its current state and run `compilemessages`, I don't get any changes on `django.mo`. So I guess this is good to go? 🤷‍♂️  I'll merge from main and see if things still pass.